helm/security-pack/Chart.yaml

```diff
@@ -4,8 +4,8 @@ description: A Helm chart for Giant Swarm's collection of open-source security t
 engine: gotpl
 home: https://github.com/giantswarm/security-pack
 sources:
-- https://github.com/giantswarm/security-pack
-version: 0.1.0
+  - https://github.com/giantswarm/security-pack
```

Comment:
This intentional?

Reply:
Looks like it's fine https://helm.sh/docs/topics/charts/#the-chartyaml-file

Reply:
No idea where that comes from tbh